manifest/manifestlist/manifestlist.go

```diff
@@ -10,28 +10,44 @@ import (
 	"github.com/docker/distribution/manifest"
 )
 
-// MediaTypeManifestList specifies the mediaType for manifest lists.
-const MediaTypeManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+const (
+	// MediaTypeManifestList specifies the mediaType for manifest lists.
+	MediaTypeManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+
+	// MediaTypeOCIManifestList specifies the mediaType for OCI compliant
+	// manifest lists.
+	MediaTypeOCIManifestList = "application/vnd.oci.image.manifest.list.v1+json"
+
+	// DefaultMediaTypeManifestList is the default choice of manifest list type
+	// when marshalling/serving content
+	DefaultMediaTypeManifestList = MediaTypeOCIManifestList
```

Comment:
Why would this be the default? The registry doesn't decide the serialization. It is up to the client.